localstripe/resources.py

```diff
@@ -2280,6 +2371,18 @@ def __init__(self, charge=None, amount=None, metadata=None, **kwargs):
         if self.amount is None:
             self.amount = charge_obj.amount
 
+    @classmethod
+    def _api_create(cls, **data):
+        obj = super()._api_create(**data)
+        txn = BalanceTransaction(amount=-obj.amount,
+                                 currency=obj.currency,
+                                 description='REFUND FOR CHARGE',
+                                 exchange_rate=1.0,
+                                 reporting_category='refund',
+                                 source=obj.id, type='refund')
+        obj.balance_transaction = txn.id
```

Comment:
Usually the creation of related entities is rather done inside `__init__()`. Is it possible to do the same here?

It could go inside a block `if self.status == 'succeeded'`, for clarity (even if it's the only refund status implemented for now).

Reply:
Done.